README.md

```diff
@@ -1 +1,42 @@
-# test-automatiom-testNG
\ No newline at end of file
+# TestNG example
+
+Application model which describes the behavior of running the test with gradle's help
+
+## Getting Started
+
+## Running
+
+1. Open project;
+2. Select the Run Configuration;
+3. Wait.
+
+## Testing
+
+Please use next rules for test the app:
+
+1 -
+``
+SmokeTests or EOSTests
+``
+
+2 -
+``
+run or debug
+``
+## Built With
+
+* [TestNG](http://testng.org/doc/) - Used to testing framework
```

Comment:
would be better to use "as a testing framework" instead of "to testing framework"